api_server.py

```
@@ -80,6 +94,35 @@ def get_crew_list() -> List[Dict]:
         return [dict(r) for r in rows]
 
 
+def format_availability_data(data: Any, now: datetime) -> Dict:
```

Ruff (0.15.2)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `data`

(ANN401)